uiowa/uiowa
# Adding collegiate sites to our run_first list

Resolves #5628 

# How to test

- checkout main, run `ddev composer install`
- `ddev blt dsa` - make sure your list contains at least one of the newly added college sites and at least one site not on the list but before the college site alphabetically (eg housing and tippie, diversity and education, etc). You can also add a site already in the run_first list, like admissions.uiowa.edu. 
- `ddev blt auda` - see that the sites run in alphabetical order (besides the five sites already in the run_first list). 
- checkout this branch (`run_first_college_sites`)
- run `ddev blt auda` again - this time the college sites should run ahead of any sites not on the run_first list. (eg tippie runs ahead of housing, education ahead of diversity).

Question: does this need a scheduled deployment?
